extension/linkify-urls-in-code.js

```diff
@@ -14,7 +14,7 @@ window.linkifyURLsInCode = (() => {
 	const linkifyURL = url => `<a href="${url}" ${commonURLAttrs}>${url}</a>`;
 
 	const hasIssue = text => issueRegex.test(text);
-	const hasURL = text => URLRegex.test(text);
+	const hasURL = text => new RegExp(URLRegex).test(text);
```

Comment:
Ah, that's because when you add `g` flag, the regex becomes stateful, so each call changes its result position. Just make the regex a function instead, so we get a fresh regex each time.

```diff
- const URLRegex = /(http(s)?(:\/\/))(www\.)?[a-zA-Z0-9-_.]+(\.[a-zA-Z0-9]{2,})([-a-zA-Z0-9:%_+.~#?&//=]*)/g;
+ const URLRegex = () => /(http(s)?(:\/\/))(www\.)?[a-zA-Z0-9-_.]+(\.[a-zA-Z0-9]{2,})([-a-zA-Z0-9:%_+.~#?&//=]*)/g;
```